runner/runner.go

```diff
@@ -232,13 +319,13 @@ func (r *Runner) Stop() error {
 	r.mux.Lock()
 	defer r.mux.Unlock()
 
-	for _, repo := range r.repositories {
+	for _, repo := range r.poolManagerCtrl.GetRepoPoolManagers() {
 		if err := repo.Stop(); err != nil {
 			return errors.Wrap(err, "stopping repo pool manager")
 		}
 	}
 
-	for _, org := range r.organizations {
+	for _, org := range r.poolManagerCtrl.GetOrgPoolManagers() {
```

Comment:
```GetOrgPoolManagers()``` should also return an error, unless it's absolutely guaranteed to never fail under any circumstances.

Reply:
Same comment as previously.